jenkins/Jenkinsfile

```diff
@@ -136,6 +136,7 @@ pipeline {
     string(name: "terra_jupyter_python", defaultValue: "0.0.6")
     string(name: "terra_jupyter_bioconductor", defaultValue: "0.0.9")
     string(name: "terra_jupyter_gatk", defaultValue: "0.0.8")
+    string(name: "terra_jupyter_aou", defaultValue: "0.0.1")
```

Comment:
If I'm understanding correctly these params are not used by the default case -- instead this script tries to parse https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/resources/reference.conf to find the versions to pull. I wonder if we should update this conf and add a test case for the AoU image similar to https://github.com/DataBiosphere/leonardo/blob/develop/automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookGATKSpec.scala

Reply:
Yeah, a test isn't strictly necessary, but I think it would be good in this PR or a follow-up one. A conf entry is preferred to updating the version in `prepare_custom_leonardo_gce_image.sh` each time as then the PR updating the version will automatically test the new image for you

Reply:
> A conf entry is preferred to updating the version in prepare_custom_leonardo_gce_image.sh each time

Asking to make sure I'm not missing anything: The conf entry is what determines how the corresponding image's version will be updated in `prepare_custom_leonardo_gce_image.sh`, no? In other words, `prepare_custom_leonardo_gce_image.sh` **is** updated each time.